cpp/test/benchmark.cc

```diff
@@ -203,37 +285,47 @@ void BenchmarkSimpleMrd(double scale = 1) {
       });
 }
 
+#define FOO(x)
```

Comment:
??

Reply:
From a foolish moment where I though "maybe I should use a macro".